lib/suse/connect/client.rb

```diff
@@ -29,6 +29,7 @@ def register!
         System.add_service(service)
         Zypper.install_release_package(product.identifier) if @config.product
         print_success_message product
+        run_post_register_scripts product
```

Comment:
When the registration happens through YaST, and not with the SUSEConnect CLI, it goes `SUSE::Connect::YaST::activate_product() -> Client.activate_product()` and will not call `run_post_register_scripts`